kvbc/src/replica_state_sync_imp.cpp

```diff
@@ -45,8 +46,8 @@ uint64_t ReplicaStateSyncImp::execute(logging::Logger& logger,
     }
     // SBFT State Metadata is not in sync with the Blockchain State.
     // Remove blocks which sequence number is greater than lastExecutedSeqNum.
-    if (removedBlocksNum == 1) {
-      std::string error = " Detected more than one block needs to be deleted from the blockchain - unsupported";
+    if (removedBlocksNum >= maxNumOfBlocksToDelete) {
```

Comment:
shouldn't we have strict inequality `>` here?

Reply:
We are checking how many blocks we have ALREADY removed, so it should be not greater than maxNumOfBlocksToDelete.

Reply:
thanks for the clarification!